src/internal.c

```diff
@@ -18818,6 +18818,9 @@ static int checkDTLS13AEADFailLimit(byte bulk_cipher_algorithm,
             }
             break;
 #endif
+        case wolfssl_cipher_null:
+            /* No encryption being done. The MAC failed must have failed. */
```

Comment:
is the comment missing a not?

Reply:
Updating to `No encryption being done. The MAC check must have failed.`